Lundalogik/lime-elements
# Text editor emit value as markdown

Fixes Lundalogik/crm-feature#4075

How to test:
I would, in the `limel-example-prosemirror-adapter-basic` example, pass in a content type and see what value gets emitted. I would also pass in a value there to see if it gets parsed correctly depending on the content type.

## Review:

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:

Linux:

macOS:

Mobile:
